force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +362,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer wordIndex = -1;
 
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+            for ( Integer i = 0; i < words.Size(); i++){
+                if (words.get(i).equals(wordToFind)){
+                    wordIndex = i;
+                    break;
+                }
+            }
+
+        return wordIndex; // Replace null with the variable you used to store the result
```

Comment:
great job breaking the loop. Lots of students miss that. My only suggestion would be to skip creating variables you don't need unless they help make the code more readable. My solution is below. I don't think it looses readability. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
 ```